rollup.config.ts

```diff
@@ -194,7 +194,7 @@ function umdDev({
     output: {
       format: 'umd',
       sourcemap: true,
-      file: `${packageDir}/build/umd/index.development.js`,
+      file: `${packageDir}/build/umd/${outputFile}.development.js`,
```

Comment:
Since all UMD builds were generating an output file with the `index` name, this change will only affect the output of the `react-query-devtools` package